spotify-controller.el

```diff
@@ -185,13 +208,18 @@ current repeating state."
   (interactive)
   (spotify-apply "player-pause"))
 
+(defun spotify-device-volume-up ()
+  "Increase the volume by 10 percent for the active device."
+  (interactive)
+  (spotify-apply "set-volume"))
+
```

Comment:
I don't recall having such function for increasing the volume in the current library. Maybe it's related to the code you are still working on?